rewriter/Prop.cc

```diff
@@ -414,8 +443,12 @@ void Prop::run(core::MutableContext ctx, ast::ClassDef *klass) {
         if (!propInfo.has_value()) {
             continue;
         }
-        auto nodes = processProp(ctx, propInfo.value(), forTStruct);
-        ENFORCE(!nodes.empty(), "if parseProp completed successfully, processProp must complete too");
+        auto processed = processProp(ctx, propInfo.value(), forTStruct);
+        ENFORCE(!processed.empty(), "if parseProp completed successfully, processProp must complete too");
+
+        vector<unique_ptr<ast::Expression>> nodes;
+        nodes.emplace_back(ensureWithoutAccessors(propInfo.value(), send));
```

Comment:
This line has the effect of keeping the prop around in the body (but we make sure it has an extra `without_accessors: true` argument).